mpconstants/countries.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf8 -*-
```

Comment:
Everywhere in zamboni this is `# -*- coding: utf-8 -*-` (with the hyphen). Does it matter?